papaparse.js

```diff
@@ -484,6 +484,10 @@ License: MIT
 		}
 	}
 
+	function customSplice(inputArray, startIndex, count) {
```

Comment:
Why is required? It will be better to just use the slice directly instead of a custom function